keep/providers/jiraonprem_provider/jiraonprem_provider.py

```diff
@@ -414,6 +502,26 @@ def _notify(
             labels = json.loads(labels.replace("'", '"'))
         try:
             self.logger.info("Notifying jira...")
+
+            if issue_id:
+                result = self.__update_issue(
+                    issue_id=issue_id,
+                    summary=summary,
+                    description=description,
+                    labels=labels,
+                    components=components,
+                    custom_fields=custom_fields,
+                    priority=priority,
+                    **kwargs,
+                )
+
+                issue_key = self._extract_issue_key_from_issue_id(issue_id)
+
+                result["ticket_url"] = f"{self.jira_host}/browse/{issue_key}"
+
+                self.logger.info("Notified jira!")
```

Comment:
logs need to indicate this was an update notification to jira.
use `extra={}` or change log text